src/lib_json/json_value.cpp

```diff
@@ -229,10 +247,10 @@ Value::CZString::CZString(char const* str, unsigned ulength, DuplicationPolicy a
   storage_.length_ = ulength & 0x3FFFFFFF;
 }
 
-Value::CZString::CZString(const CZString& other)
-    : cstr_(other.storage_.policy_ != noDuplication && other.cstr_ != 0
-                ? duplicateStringValue(other.cstr_, other.storage_.length_)
-                : other.cstr_) {
+Value::CZString::CZString(const CZString& other) {
+  cstr_ = (other.storage_.policy_ != noDuplication && other.cstr_ != 0
```

Comment:
I was wondering about this change. In general, it's slightly faster to a initialization list, but not for built-in types like pointers. So I don't care. Just curious.

* http://stackoverflow.com/questions/13214241/c-constructors-vs-initialization-lists-speed-comparison